sys-cluster/kube-router/kube-router-0.3.1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+EGO_PN="github.com/cloudnativelabs/${PN}"
+
+inherit golang-build golang-vcs-snapshot
+
+ARCHIVE_URI="https://github.com/cloudnativelabs/kube-router/archive/v${PV}.tar.gz -> kube-router-${PV}.tar.gz"
+KEYWORDS="~amd64"
+
+DESCRIPTION="Kube-router, a turnkey solution for Kubernetes networking"
```

Comment:
`A turnkey solution for Kubernetes networking` should be enough for `DESCRIPTION`